xkcd/cmd/main.go

```diff
@@ -0,0 +1,42 @@
+// xkcd is an xkcd offline indexing and quering tool.
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/vespian/go-exercises/xkcd/pkg/cmdline"
```

Comment:
The golden standard in golang is always the go stdlib. "show me where they follow this pattern in stdlib" is the strongest argument against a pattern in the wild.

I always say: start with a main(). If the main gets big, extract into functions. When the functions get useful to someone else or the package gets really just too big, extract into packages.